File: 3rdPeriod/ObjectOrientedProgramming/Exercicio1/ExercicioPython/Ex5.py
import MyMath as m

class main():
    try:
        n = int(input("Insira um número inteiro: "))
    except ValueError():
        print("Número inválido! ")
        exit()
    original = n
    while(m.isPrime(n) == False):
        n-=1
    print("O maior número primo abaixo de {} é {}".format(original, n))    

File: 3rdPeriod/ObjectOrientedProgramming/Exercicio1/src/Ex5.java
public class Ex5 {
    public static void main(String[] args){
        int n;
        try {
            n = EntradaTeclado.leInt();
        }catch(Exception IOException) {
            System.out.println("Insira um número inteiro");
            return;
        }
        int original = n;
        while(!MyMath.isPrime(n)) n--;
        System.out.print("O primo mais próximo de " + original + " é " + n);
    }
}
